model/config.go

```diff
@@ -2348,8 +2364,9 @@ func ConfigFromJson(data io.Reader) *Config {
 	return o
 }
 
+// isNew guesses if the current config file is a new one or we are making an upgrade based on SiteURL being different from default or nil
```

Comment:
Could we rename the func/flag to be `isUpdate` and write this as follows?
```suggestion
// isUpdate detects a pre-existing config based on whether SiteURL has been changed
```